src/coreComponents/constitutive/unitTests/testCompositionalProperties.cpp

```diff
@@ -195,11 +197,9 @@ class CompositionalPropertiesTestDataTestFixture : public ::testing::TestWithPar
   real64 computeMolarDensity( real64 const pressure, real64 const temperature,
                               arrayView1d< real64 const > const & composition ) const
   {
-    auto criticalPressure = m_fluid->getCriticalPressure();
-    auto criticalTemperature = m_fluid->getCriticalTemperature();
-    auto acentricFactor = m_fluid->getAcentricFactor();
-    auto volumeShift = m_fluid->getVolumeShift();
-    real64 const binaryInteractionCoefficients = 0.0;
+    auto const componentProperties = this->m_fluid->createKernelWrapper();
```

Comment:
this `createKernelWrapper` looks a bit ambiguous, this is only in tests, right?

Reply:
Yes it's not really a kernel wrapper.